src/Stratis.Bitcoin/P2P/Peer/NetworkPeerDisposer.cs

```diff
@@ -36,7 +36,7 @@ public class NetworkPeerDisposer : IDisposable
         private readonly AsyncQueue<INetworkPeer> peersToDispose;
 
         /// <summary>Mapping of connected peers by their connection ID.</summary>
-        private readonly ConcurrentDictionary<int, INetworkPeer> connectedPeers;
+        public ConcurrentDictionary<int, INetworkPeer> connectedPeers { get; }
```

Comment:
Instead or exposing the internal dictionary perhaps better create a method that return the count of outbound peers?